src/main/java/com/hcl/appscan/jenkins/plugin/scanModes/ScanModeFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.hcl.appscan.jenkins.plugin.scanModes;
+
+public class ScanModeFactory {
+
+    public static ScanMode getScanTypeImplementation(String scanType, String loginType, String userName, String password, String trafficFile, String exploreData) {
```

Comment:
Not a must, but this method would be better if it were called "getScanMode", since this is a ScanModeFactory. It's not clear what is meant by "getScanTypeImplementation".

Reply:
Okay, I will rename it to "getScanMode".